server/templates/email/media-request/html.pug

```diff
@@ -25,93 +25,63 @@ head
       font-family: 'Segoe UI', sans-serif;
       mso-line-height-rule: exactly;
       }
+      html {
+      font-size: calc(16px + 6 * ((100vw - 600px) / 680));
+      }
   style.
+    html {
+    font-family: 'Inter', Arial, Sans-Serif;
+    }
+    .title:hover * {
+    text-decoration: underline;
+    }
     @media (max-width: 600px) {
     .sm-w-full {
     width: 100% !important;
     }
+    .incsize {
+    font-size: larger;
     }
```

Comment:
What is the purpose of this, if you're setting varying `font-size`s in the parent elements anyway?

Reply:
This is used as a trigger for mobile devices to increase the text-size for timestamps, title, and extraInfo (which all have varying sizes). There's probably a better way to do this, though. All the font-sizes are referring to the 'root em' sizes anyways, so maybe I can test it by changing the font-size at html instead.

Reply:
Rescaled the fonts, and switched them to 'em' and defined the table as parent element to scale off of. @media CSS query was broken due to white spacing and was resolved. Removed all instances of 'incsize' classes.